Sources/swift-format/Frontend/Frontend.swift

```diff
@@ -268,6 +270,64 @@ class Frontend {
       }
     }
 
+    // Load global configuration file
+    // First URLs are created, then they are queried. First match is loaded
+    var configLocations: [URL?] = []
```

Comment:
Sorry, I only now noticed: Do we ever add `nil` to this array? I don’t think so and we wouldn’t need to do `case var location?` in the loop below if we don’t allow optionals in here.